diff --git a/node_modules/react-native-http-bridge-refurbished/android/build.gradle b/node_modules/react-native-http-bridge-refurbished/android/build.gradle
index ce940df..2f63a8f 100644
--- a/node_modules/react-native-http-bridge-refurbished/android/build.gradle
+++ b/node_modules/react-native-http-bridge-refurbished/android/build.gradle
@@ -37,6 +37,5 @@ android {
 
 dependencies {
     implementation 'com.facebook.react:react-native:+'
-    implementation 'com.google.android.gms:play-services-gcm:16.1.0'
     implementation 'org.nanohttpd:nanohttpd:2.3.1'
 }
